### DISCUSSION TITLE
Export plot

### DISCUSSION THREAD

**Discussion by vfreysz** [Avaiga/taipy#103] [Q&A] [answered]
Hello,

how can i export a plot as a svg ?

**Comment by FlorianJacta** [comment]
Hi!

You can, for now, download it as 'png'. You can click on 'Download plot as a png' among the little buttons on the top-right of a plot.

Taipy is based on top of Plotly for charts, so there might be a way to download it as 'svg' not implemented by default with Taipy.

**Comment by vfreysz** [comment]
Thank you for the answer. 
Is there anyway for me to access plotly running behind and code the export in javascript:

https://plotly.com/javascript/static-image-export/

I have one workaround that is using plotly with python and reproduce the chart i am displaying but it is not very efficient.

**Comment by FlorianJacta** [comment, marked as answer]
@vfreysz You can change the configuration to download an 'svg' like you would using [Plotly Javascript](https://plotly.com/javascript/configuration-options/#customize-download-plot-options). Taipy provides the API called _plot_config_ to change the Plotly configuration. Other parameters can also be changed that are described in the [Plotly configuration page](https://plotly.com/javascript/configuration-options/).

```python
from taipy import Gui

x = [1, 2, 3, 4, 5]
y = [2, 4, 1, 6, 3]
data = {'x': x, 'y': y}

# the format can be png, svg, jpeg, webp
config = {
    "toImageButtonOptions": {"format": 'svg'},  
}

Gui("<|{data}|chart|plot_config={config}|>").run()
```
